commands/economy/setpoint.js

```diff
@@ -0,0 +1,116 @@
+const Discord = require("discord.js")
+const { COLOR } = process.env;
+
+module.exports = {
+  name:"setpoint",
+  category:"economy",
+  description:"Set user point add or remove",
+  usage:'<user> <add or remove> <nominal> <reason>',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mhazm/Saino-sama/pullRequest?prid=7538397)
```suggestion
  usage:"<user> <add or remove> <nominal> <reason>",
```